test cases/rust/18 proc-macro/meson.build

```diff
@@ -8,6 +8,7 @@ pm = shared_library(
   'proc_macro_examples',
```

Comment:
A couple of lines above, why is this test disabled on non-Linux, how does it fail? Theoretically there shouldn't be anything special required.

Reply:
@sdroege some of the tests are disabled on certain platforms because toolchains might not be setup on those platforms (work needed) or tests can take too long. In this case, you are probably running into the second option. Azure CI takes an ungodly amount of time and will timeout frequently.

Reply:
It could very well be historical reasons that have been fixed since then. Feel free to open a separate PR that unskip them and see how it goes on the CI.